action.yml

```
@@ -33,5 +36,17 @@ runs:
         # the request metadata to pkg.julialang.org when installing
         # packages via `Pkg.test`.
         JULIA_PKG_SERVER: ""
-    - run: julia --color=yes --check-bounds=yes --inline=${{ inputs.inline }} --project -e 'using Pkg; Pkg.test(coverage=${{ inputs.coverage }})'
+    - run: |
+        julia --color=yes --check-bounds=yes --inline=${{ inputs.inline }} --project -e '
+          using Pkg
+          kwargs = Dict{Symbol, Any}()
+          kwargs[:coverage] = ${{ inputs.coverage }
+          if ${{ inputs.force_latest_compat }}
+              if VERSION > VersionNumber(1, 6, typemax(UInt32)) # excludes 1.6, includes 1.7-DEV, includes 1.7
```

```suggestion
              if VERSION >= v"1.7.0-" # excludes 1.6, includes 1.7-DEV, includes 1.7
```

I think this is equivalent but slightly easier to read:

```julia
julia> v = v"1.7.0-"
v"1.7.0-"

julia> v"1.6.4294967295" >= v
false

julia> v"1.7-DEV" >= v
true

julia> v"1.7.0-" >= v
true

julia> v"1.7.0-DEV" >= v
true

julia> v"1.7.0-alpha" >= v
true

julia> v"1.7.0-rc1" >= v
true

julia> v"1.7.0-0" >= v
true
```